src/CommunityToolkit.Maui/Animations/RotateAnimation.shared.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace CommunityToolkit.Maui.Animations;
+
+/// <summary>
+/// Animation that will rotate the supplied view by the specified <see cref="Rotation"/>.
+/// </summary>
+public class RotateAnimation : BaseAnimation
+{
+	/// <summary>
+	/// Backing BindableProperty for the <see cref="Rotation"/> property.
+	/// </summary>
+	public static readonly BindableProperty RotationProperty =
+		BindableProperty.Create(
+			nameof(Rotation),
+			typeof(double),
+			typeof(RotateAnimation),
+			180.0,
+			BindingMode.TwoWay,
+			defaultValueCreator: GetDefaultRotationProperty);
+
+	/// <summary>
+	/// Gets or sets the rotation used by the animation.
+	/// </summary>
+	public double Rotation
```

Comment:
maybe name it Angle?

Reply:
This is the property name that was used in XCT and it relates to the property that it is animating (`VisualElement.Rotation`). I would like to investigate more around our use of `RotateTo` vs `RelRotateTo` though.

Reply:
let's keep Rotation: https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.visualelement.rotation?view=net-maui-7.0#microsoft-maui-controls-visualelement-rotation